src/history.ts

```diff
@@ -0,0 +1,45 @@
+import { randomUUID } from 'node:crypto';
+import { type HistoryIssue } from './shared';
+
+export const history = new Array<HistoryIssue>();
+
+export const addNewHistoryIssue = ({ action, issue, replacer }: Omit<HistoryIssue, 'id'>) => {
+	if (history.length >= 100) {
+		history.shift();
+	}
```

Comment:
100 concerns me because I would think that would totally flood the user's console? Also this wouldn't really alleviate the problem we discussed over discord since it's still a very large amount of issues. I was thinking 5 to 10. If we do 100, maybe consider pagination?

Reply:
I replaced it with just 10 on last commit :)